packages/react/src/components/Tooltip/index.tsx

```diff
@@ -80,6 +80,25 @@ export default function Tooltip({
     }
   }, [popperPlacement]);
 
+  useEffect(() => {
+    const handleEscape = (event: KeyboardEvent) => {
+      if (event.key === 'Escape') {
```

Comment:
In Edge, this is `Esc`. If we support Edge, we should check for both:

```suggestion
      if (event.key === 'Escape' || event.key === 'Esc') {
```

Reply:
Good call. We should, I'll implement a graceful fallback.